src/components/Auth/Authenticate.tsx

```diff
@@ -0,0 +1,254 @@
+import careConfig from "@careConfig";
+import { t } from "i18next";
+import { Link } from "raviger";
+import { useState } from "react";
+
+import Card from "@/CAREUI/display/Card";
+import CareIcon from "@/CAREUI/icons/CareIcon";
+
+import { Button } from "@/components/ui/button";
+import { CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+export const Authenticate = () => {
+  const { urls, stateLogo, customLogo, customLogoAlt } = careConfig;
+  const customDescriptionHtml = __CUSTOM_DESCRIPTION_HTML__;
+  const logos = [stateLogo, customLogo].filter(
+    (logo) => logo?.light || logo?.dark,
+  );
+  const [recoveryModal, setRecoveryModal] = useState(false);
+
+  const accesWays: string[] = [
+    "OTP via SMS",
+    "OTP via Email",
```

Comment:
Remove these two methods for now(V1) we are only implementing the TOTP with recovery codes as mentioned in the [issue](https://github.com/ohcnetwork/care_fe/issues/9456#:~:text=We%20decided%20to%20implement%20the%20TOTP%20with%20backup%20codes%20and%20eventually%20add%20support%20for%20other%20methods%20in%20future), we can add the remaining two when we support them.